### PR TITLE
feat: disable sponsorblock on shorts

### PR DESCRIPTION
**Pull request by OxrxL**

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java [hunk 1]
```diff
@@ -108,6 +108,7 @@ public enum SettingsEnum {
     SB_SHOW_BROWSER_BUTTON("sb-browser-button", false, SharedPrefHelper.SharedPrefNames.SPONSOR_BLOCK, ReturnType.BOOLEAN),
     SB_API_URL("sb-api-url", "https://sponsor.ajay.app/api/", SharedPrefHelper.SharedPrefNames.SPONSOR_BLOCK, ReturnType.STRING);
 
+    public static boolean SB_SHORTS_ENABLED = false;
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java:111]: Make sure to also adjust this in the [related patch](https://github.com/revanced/revanced-patches/pull/439).

```suggestion
    public static boolean shorts_playing = false;
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Comment by OxrxL** [comment]
> There are a lot of changes, that were not necessary. The `sponsorblock` patch only injects couple of calls to methods into the application, only these methods should be blocked.

I need to know which are these methods, otherwise it will become quite difficult for me to find out

**Comment by oSumAtrIX** [comment]
Sure, all calls to the methods in the integrations can be found [here](https://github.com/revanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt). Simply `CTRL+F` and search for `integrations/` to get a good understanding.

**Comment by OxrxL** [comment]
> Sure, all calls to the methods in the integrations can be found [here](https://github.com/revanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt). Simply `CTRL+F` and search for `integrations/` to get a good understanding.

At first look I should find an unused register for each call. It does not make the operation more complex?

**Comment by OxrxL** [comment]
> Sure, all calls to the methods in the integrations can be found [here](https://github.com/revanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt). Simply `CTRL+F` and search for `integrations/` to get a good understanding.

Nevermind. I said nonsense. You were referring to the changes to be made to the integrations and not to the patch. xD

**Comment by oSumAtrIX** [comment]
No problem, yeah, simply add the condition in the integrations methods instead of patching the application.

**Comment by OxrxL** [comment]
Should be good now. For safeness, test it you too. :D

File: app/src/main/java/app/revanced/integrations/sponsorblock/SponsorBlockUtils.java [hunk 2]
```diff
@@ -403,6 +403,8 @@ public static void notifyShareBtnVisibilityChanged(View v) {
     }
 
     public static String appendTimeWithoutSegments(String totalTime) {
+        if (SettingsEnum.shorts_playing) return "";
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/sponsorblock/SponsorBlockUtils.java:406]: Should this not be:
```suggestion
        if (SettingsEnum.shorts_playing) return totalTime;
```
?

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**